src/category_theory/limits/shapes/pullbacks.lean

```diff
@@ -148,16 +178,16 @@ abbreviation pullback_cone (f : X ⟶ Z) (g : Y ⟶ Z) := cone (cospan f g)
 namespace pullback_cone
 variables {f : X ⟶ Z} {g : Y ⟶ Z}
 
-def π₁ (t : pullback_cone f g) : t.X ⟶ X := t.π.app left
-def π₂ (t : pullback_cone f g) : t.X ⟶ Y := t.π.app right
+def fst (t : pullback_cone f g) : t.X ⟶ X := t.π.app left
+def snd (t : pullback_cone f g) : t.X ⟶ Y := t.π.app right
 
-def mk {W : C} (π₁ : W ⟶ X) (π₂ : W ⟶ Y) (eq : π₁ ≫ f = π₂ ≫ g) : pullback_cone f g :=
+def mk {W : C} (fst : W ⟶ X) (snd : W ⟶ Y) (eq : fst ≫ f = snd ≫ g) : pullback_cone f g :=
 { X := W,
   π :=
-  { app := λ j, walking_cospan.cases_on j π₁ π₂ (π₁ ≫ f),
+  { app := λ j, walking_cospan.cases_on j fst snd (fst ≫ f),
     naturality' := λ j j' f, by cases f; obviously } }
 
-lemma condition (t : pullback_cone f g) : (π₁ t) ≫ f = (π₂ t) ≫ g :=
+lemma condition (t : pullback_cone f g) : (fst t) ≫ f = (snd t) ≫ g :=
```

Comment:
Isn't this usually called `w` instead of `condition`? Also, do we want a `reassoc` version of it?

Reply:
Yes to both, fixed.

Reply:
Sorry, reversing this response. The problem is that `w` is about the triangle, whose sides are two maps from the cone point, and whose base is some arrow in the index diagram. The `condition` lemma is built by gluing these triangles together, so it needs another name. `condition` is not exactly descriptive, but it's not helpful to overload `w` at this point.